client/src/pages/index.tsx

```diff
@@ -57,6 +57,10 @@ function IndexPage({
     allChallengeNode: { nodes: challengeNodes }
   }
 }: Props): JSX.Element {
+  useEffect(() => {
+    sessionStorage.setItem('canOpenModal', 'true');
+  }, []);
```

Comment:
We're still setting this. TBH, we shouldn't need to set it at all.

Reply:
So you want the developer to manually put the item "canOpenModal" in the application menu ? This just puts it there automatically saving sometime.

Reply:
The developer should manually set `canOpenModal=false` in the session store.

If this is not set, or set to any value besides false, the code should not do anything.

Reply:
Ok done